.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```

Comment:
maybe [FEATURE] or [WISH]

Reply:
I actually added that but GitHub lost it (probably because I did both at the same time).

Reply:
Fixed.